go.mod

```diff
@@ -28,19 +28,27 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/spf13/cobra v1.6.1
 	github.com/stretchr/testify v1.9.0
-	golang.org/x/crypto v0.21.0
+	golang.org/x/crypto v0.24.0
 	golang.org/x/oauth2 v0.17.0
 	gopkg.in/gomail.v2 v2.0.0-20160411212932-81ebce5c23df
 )
 
 require (
 	github.com/bits-and-blooms/bitset v1.10.0 // indirect
+	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.3.0 // indirect
```

Comment:
hmm i don't think that curve is supported based on [crypto/ecdh](https://pkg.go.dev/crypto/ecdh#PublicKey)

Reply:
imo should be fine since the import has been [around for a while though](https://github.com/lestrrat-go/jwx/blame/develop/v2/jwk/es256k.go#L7)


Additional info:

Looks like this was imported via `jwk`

`go mod why -m github.com/decred/dcrd/dcrec/secp256k1/v4`

Initial suspiscion is warranted since `dcrec` bitcoin related and supply chain attacks are not uncommon. Personally think it's fine to keep for now


Added context:

`crypto/x509` also does not [seem to support this curve x509 signing](https://github.com/lestrrat-go/jwx/pull/1117/files#diff-bbd4b6a86bc65b6ac8e79e97afc61499158edb40f7bb404c70637b46d80a7ad2R50) 

It sounds like not in stdlib[it's considered a custom curve](https://github.com/golang/go/issues/26776#issuecomment-497671740)